rules/cross-platform/execution_suspicious_jar_child_process.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious child processes of a Java Archive (JAR) file. JAR files may be used to deliver malware in order to
+evade detection.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious JAR Child Process"
+risk_score = 47
+rule_id = "8acb7614-1d92-4359-bfcf-478b6d9de150"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
```

Comment:
to reduce a bit some noisy instances (there could be more):
```
and not process.executable:"/Users/*/.sdkman/*" and not process.args:("/usr/local/*", "/Users/*/github.com/*", "/Users/*/src/*")
```

Reply:
thanks will add an FP section